js/constants.js

```diff
@@ -41,6 +41,17 @@
         correct: 'rgba(98, 252, 101, .9)',
         incorrect: 'rgba(255, 112, 102, .9)'
     }
+    
+    // filter links from string
+    root.filterLinks = string => {
+        let parts = string.split('src="http')
+        let result = parts[0]
+        parts.forEach(part => {
+            let index = part.indexOf('"')
+            let newLink = part.slice(0,index).split('/').pop()
+            result += 'src="' + newLink + part.slice(index)
+        })
```

Comment:
I don't understand what's happening here?
How does this work? It doesn't even return anything D:
Also, you don't need to initialize the `result` variable with the first element of the `parts` array, the
```
parts.forEach(_ => {
    ...
    result += ...
})
```
will already do that for you...

Reply:
Maybe you guys should take care of the filter part :P
The important part is the substitution of the answers in the old filter with the ones in the new one.

Reply:
[Discussion]
Does it make any sense to port the answers in `answer.js`?
If we could identify which question it was, we wouldn't need to fix the links/latex in the first place
I do agree we should try to clean the DB. Porting the answers seems like the best option, but if anywhere, shouldn't this be done in `scan.js` or `collector.js`?

Reply:
agreed, porting the answers shouldn't be done in the `anwers.js`
I think it makes more sense to be at `collector.js` since stuff that will be retrieved in `scan.js` should also be retrieved in `collector.js`

Reply:
I think it should done in both `collector.js` and `answer.js`.
The thing is, I wasn't sure the code worked properly, so I wanted to do a smaller test with less updates to the base before pushing the whole thing into production.

Reply:
fair enough